lib/block/block.js

```diff
@@ -62,27 +51,35 @@ Block._from = function _from(arg) {
 };
 
 /**
- * @param {String|Object} - A JSON string or object
+ * @param {String} - A JSON string
  * @returns {Object} - An object representing block data
  * @private
  */
 Block._fromJSON = function _fromJSON(data) {
```

Comment:
I think many fromJSON methods currently handle both an object and string, we should make sure these are all consistent.